packages/nuxt/src/app/components/nuxt-teleport-island-slot.ts

```diff
@@ -35,23 +35,23 @@ export default defineComponent({
     }
 
     return () => {
-      const vnodes: VNode[] = []
+      const vnodes: VNode[] = new Array(1)
```

Comment:
this array might have either 1 or 2 members - is it worth optimising it for 1?

Reply:
From the very scarce resources about the internal operations in arrays, in v8, JSC, and Bun, using `push` immediately allocates space for ~8-16 spaces, and then doubles capacity on each push where it exceeds the array length

If it ends up with 1 element, then it avoids those extra 7-15 spaces. If it ends up with 2 elements, it would allocate another single place, which means it would be left with exactly 2 spots for 2 elements, once again avoiding the extra space